cmd/kpod/load.go

```diff
@@ -101,9 +105,19 @@ func loadCmd(c *cli.Context) error {
 // loadImage loads the image from docker-archive or oci to containers-storage
 // using the pullImage function
 func loadImage(store storage.Store, opts loadOptions) error {
-	systemContext := common.GetSystemContext("")
-
 	src := dockerArchive + opts.input
```

Comment:
Nit: it would be a tiny bit more readable if the `src` assignment were closer to the first user, so that the blank line separates the options used for both attempts and the data for the individual attempts.

Reply:
(Absolutely non-blocking)

Reply:
fixed.